src/cli.ts

```diff
@@ -214,3 +227,36 @@ async function routeGenerator(args) {
     process.exit(1);
   }
 }
+
+async function generateSwaggerAndRoutes(args) {
```

Comment:
Since `generateSwaggerAndRoutes` might someday become the "critical path" for tsoa, we should probably get it plugged into the integrated tests. So can you please replace a few of the `generateRoutes` inside of `prepare.ts` with `generateSwaggerAndRoutes` so that we will know when/if `generateSwaggerAndRoutes` regresses?